assets/stylesheets/shared/_color-schemes.scss

```diff
@@ -262,7 +262,7 @@
 	--masterbar-item-new-editor-background: #{$muriel-gray-500};
 	--masterbar-item-new-editor-hover-background: #{$muriel-gray-400};
 	--masterbar-toggle-drafts-editor-background: #{$muriel-gray-400};
-	--masterbar-toggle-drafts-editor-border-color: #{$gray-lighten-20};
+	--masterbar-toggle-drafts-editor-border-color: #{--color-neutral-100};
```

Comment:
does this one work?

Reply:
Oops, no. Also this change shouldn't be in this PR. Fixed that.